openapi/definitions.yaml

```diff
@@ -637,6 +637,29 @@ ServiceTuple:
   required:
     - service_id
     - version
+UserDataProcessing:
+  type: object
+  description: |-
+    A representation of a single user data processing request
+  properties:
+    fiscal_code:
+      $ref: "#/FiscalCode"
+    choice:
+      $ref: "#/UserDataProcessingChoice"
+    status:
+      $ref: "#/UserDataProcessingStatus"
+    created_at:
+      $ref: "#/Timestamp"
+    updated_at:
+      $ref: "#/Timestamp"
+    version:
+      type: integer
+  required:
+    - fiscal_code
+    - choice
+    - status
+    - created_at
```

Comment:
it looks like the API caller has to provide a create_at value to insert a new request. is it possible to make this field computed server side ? (same for update_at)

Reply:
Yes it is possible but I' don' t know if it is mandatory to mantain the created_date generation on client side (app) for legal purpose.